kit/ChildSession.cpp

```diff
@@ -671,6 +671,15 @@ bool ChildSession::_handleInput(const char *buffer, int length)
             }
             else if (tokens[1].find(".uno:SetDocumentProperties") != std::string::npos)
```

Comment:
What is tokens[1] when this problem arises?

Maybe we should have just:

else if (tokens[1] == ".uno:SetDocumentProperties") )

here if we just care about the "bare" case that shows the dialog. (Assuming that is all we get when we launch the properties dialog) ?

Reply:
good point. KISS